docs/src/pages/customization/themes.md

```diff
@@ -136,6 +136,20 @@ Lacking imagination? The Material Design team has built an awesome [palette conf
 ### Dark/light theme
 
 You can make the theme dark by setting `type` to `dark`.
+It's a single property value change.
+Internally, it's modifying the value of the following keys:
```

Comment:
While it's only a single property value change, internally it modifies the value of the following keys: